x-pack/legacy/plugins/fleet/public/utils/date.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment';
+
+export function formatDate(timestamp: string) {
```

Comment:
[FormattedDate](https://github.com/formatjs/react-intl/blob/master/docs/Components.md#formatteddate) can be used here (this available as an export from `@kbn/i18n/react`, but not listed on that package's readme)